### PR TITLE
Display Course Credit Categories

### PR DESCRIPTION
**Pull request by frags51**
The show GPA button now also displays the credits of some categories that you have done (eg LA/CA, Free Electives, Basic Sciences, etc) - useful for BTech people.

### CODE REVIEW

File: gpameter.user.js [hunk 1]
```diff
@@ -68,20 +82,51 @@ show_total_gpa = function(){
...
+		</ul>');
 }
 if (!(typeof unsafeWindow === 'undefined')) {
   unsafeWindow.show_total_gpa = show_total_gpa;
   unsafeWindow.exclude_list = exclude_list;
   unsafeWindow.add_checkbox = add_checkbox;
   unsafeWindow.grade_values = grade_values;
 }
-$('#studentCourseSearch').before('<input id="gpa_button" class="btn" value="Show Gpa" style="width:75px; margin-right:10px; opacity:1;" type="button" onclick="show_total_gpa();"></input>');
+$('#studentCourseSearch').before('<input id="gpa_button" class="btn" value="Show Gpa" style="width:75px; margin-right:10px; opacity:1;" type="button" onclick="show_total_gpa();"></input>');
```
Comment (AgamAgarwal) [inline gpameter.user.js]: Could you add back the newline here?

Comment (frags51) [reply gpameter.user.js]: Will do.

**Comment by frags51** [comment]
@AgamAgarwal : Resolved this (finally : P). Please merge.

**Comment by frags51** [comment]
Removed. Repl.it had slyly added that file.

**Comment by frags51** [comment]
@AgamAgarwal Gentle reminder :D

**Pull request merged**